jdi-light-html-tests/src/main/java/io/github/com/sections/ContactFormUI.java

```diff
@@ -0,0 +1,28 @@
+package io.github.com.sections;
+
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.composite.Form;
+import com.epam.jdi.light.elements.interfaces.complex.IsCombobox;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.Checkbox;
+import com.epam.jdi.light.ui.html.elements.common.TextArea;
+import com.epam.jdi.light.ui.html.elements.common.TextField;
+import io.github.com.entities.Contacts;
+
+public class ContactFormUI extends Form<Contacts> {
+	@UI("#name") TextField name;
+	@UI("#last-name") TextField lastName;
+	@UI("#position") TextField position;
+	@UI("#passport-number") TextField passportNumber;
+	@UI("#passport-seria") TextField passportSeria;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4997679)